### ISSUE TITLE
Anouncement:  I'm developing a Record/Replay tool

### ISSUE THREAD

**Issue by beuaaa** [pywinauto/pywinauto#956]
Hello, I have developed a Pywinauto "Record/Replay" tool. There is still a lot of work to do but I would like to have first opinions and I'm looking for beta testers.

Here is a small video presentation:  [Youtube presentation](https://www.youtube.com/watch?v=-7W-rOvUjdE&list=PLV9GWm_Y6wCAW5aaDGSNJwwenmRk6dI-u)

I just realized that pywinauto_recorder.exe doesn't work on all machines (I'll fix this problem later). So to test it I advise you to download the ZIP archive of the clone.:
https://github.com/beuaaa/pywinauto_recorder/archive/master.zip

**Comment by airelil** [comment]
Really good work! Looks like you're quite invested into it. Do you mind providing a bit more details: what's supported for now and any limititaions?

Not sure if you're aware of a separate dev branch we have: https://github.com/pywinauto/pywinauto/commits/dev-script-generator . It contains the code for ongoing work on pywinauto internal recorder. Unfortunately, I wasn't involved into it much. Mostly @vasily-v-ryabov. The branch is also updated with all the latest dev code from our "next-to-release" atspi branch.

**Comment by beuaaa** [comment]
Thank you very much for your encouragement. Like Vasily Ryabov, it's a hobby. So I have to balance free time after work and with my family.

This is my first Open Source Python project and I discover everything: GitHub, Automation, Pypi, etc... So I have a lot of things to learn. It's part of my pleasure. 

The limitations of my tool are the same as Pywinauto's and also for the moment it only uses UIA backend.

Vasily Ryabov told me about the pywinauto internal recorder project. I'm going to throw in following that. It's certainly much more complex than my little tool. If I can, I will help him to develop his project but I'll develop mine first, there are so many things to do!

To go faster I will not be against the help of more experienced people.
For example I realized that [Pywinauto_recorder.exe doesn't work on all machines](https://github.com/beuaaa/pywinauto_recorder/issues/9) @airelil If you could help me with this issue I would be very grateful. I think it has something to do with this [Pywinauto issue ...](https://github.com/pywinauto/pywinauto/issues/911) but i'am not sure.

When I opened the problem I also realized that `pip install pywinauto_recorder` wasn't working so I quickly closed the problem. Now I fixed it so I'm reopening it to give time to people interested to discover my project.

**Comment by airelil** [comment]
Yep. It sounds like another problem with the comtypes cache. Did you try to solve it the same way, by re-generating the cached COM code?

**Comment by vasily-v-ryabov** [comment]
Hi @beuaaa I've posted my thoughts about comtypes usage to the recorder issue. This is really big job done. It involves mini-language for `left_click(...)` parameter. I've never thought previously about the idea to use recorder code in the recorded script. Though this is one more dependency.

I'm also interested to take a look into https://github.com/beuaaa/overlay_arrows_and_more code for the cursor control functions.

Problems with packing Python code into single executable are pretty wide. I have no short-term or mid-term plans to dive into this direction. But doing it properly is one of the fundamental tasks for any recorder or UI inspector like https://github.com/pywinauto/py_inspect .

**Comment by jjbright** [comment]
I tried with Unified Service Desk 4.1, the application is crashing while recording.

I see these files 
![image](https://user-images.githubusercontent.com/55202374/86800111-4f828980-c090-11ea-9833-2f555140cb6b.png)

Inside the file I've only the content below.
```
# encoding: utf-8

import os, sys
script_dir = os.path.dirname(__file__)
sys.path.append(script_dir)
from pywinauto_recorder.player import *
```

**Comment by beuaaa** [comment]
@jjbright Thank you very much for sharing this first issue. I've added your issue in Pywinauto recorder issue list
[your issue in Pywinauto recorder issue list](https://github.com/beuaaa/pywinauto_recorder/issues/11)

Could you describe the problem [here](https://github.com/beuaaa/pywinauto_recorder/issues/11) in more detail?

**Comment by danielxiaozhiming** [comment]
record click combobox is click a button and them select a ListItem, how to replay it and transform it to combobox.select()
